services/bitbucket/bitbucket-pull-request.service.js

```diff
@@ -60,6 +103,12 @@ function pullRequestClassGenerator(raw) {
           pattern: ':user/:repo',
           staticExample: this.render({ prs: 22 }),
         },
+        {
+          title: 'Bitbucket Server open pull requests',
+          exampleUrl: 'https/bitbucket.mydomain.net/foo/bar',
```

Comment:
I know `exampleUrl` usage was already in here, but would you mind updating this (and the above) to use `namedParams` instead? It's a change we're slowly making across the board in all services for performance, etc. improvements. Basically just swap out the `exampleUrl` key and replace it with:

```js
namedParams: {
  proto: 'https',
  hostAndPath: 'bitbucket.mydomain.net',
  user: 'foo',
  repo: 'bar',
}
```